app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -2,6 +2,7 @@ module Katello
   class Api::V2::ContentViewFilterRulesController < Api::V2::ApiController
     before_filter :find_filter
     before_filter :find_rule, :except => [:index, :create]
+    before_filter :create_rules, only: [:create]
```

Comment:
Yea, I like the hash rocket, but we'd get used to it if we settled on the new style